utils/admin_console_plugin_index.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+function extractTextsFromPlugin(plugin) {
+    const texts = [];
+    if (plugin.name) {
+        texts.push(plugin.name);
+    }
+    if (plugin.id) {
+        texts.push(plugin.id);
+    }
+    if (plugin.settings_schema) {
+        if (plugin.settings_schema.footer) {
+            texts.push(plugin.settings_schema.footer);
+        }
+        if (plugin.settings_schema.header) {
+            texts.push(plugin.settings_schema.header);
+        }
+
+        if (plugin.settings_schema.settings) {
+            for (const setting of Object.values(plugin.settings_schema.settings)) {
+                if (setting.label) {
+                    texts.push(setting.label);
+                }
+                if (setting.display_name) {
+                    texts.push(setting.display_name);
+                }
+                if (setting.help_text) {
+                    texts.push(setting.help_text);
```

Comment:
The `help_text` contains markdown that can include urls that are not displayed and show false results as experienced in the preliminary testing. We need to strip this out before adding to the searchable texts, you can probably use [this util markdown](https://github.com/mattermost/mattermost-webapp/blob/c7493e03dbe40e56a1e4a7e6ca7138ea3d9284df/utils/markdown/index.js#L29) `stripMarkdown` function

Reply:
done 👍